airbyte-server/src/test/java/io/airbyte/server/handlers/MigrationHandlerTest.java

```diff
@@ -0,0 +1,141 @@
+
+  @Test
+  void testConfigMigration() throws ConfigNotFoundException, IOException, JsonValidationException {
+    final StandardWorkspace workspace = generateWorkspace();
+    final StandardSourceDefinition standardSource = SourceDefinitionHelpers.generateSource();
+    final StandardDestinationDefinition standardDestination = DestinationDefinitionHelpers.generateDestination();
+    final SourceConnection sourceConnection1 = SourceHelpers.generateSource(standardSource.getSourceDefinitionId());
+    final SourceConnection sourceConnection2 = SourceHelpers.generateSource(standardSource.getSourceDefinitionId());
+    final DestinationConnection destinationConnection = DestinationHelpers.generateDestination(standardDestination.getDestinationDefinitionId());
+    final StandardSync destinationSync = ConnectionHelpers.generateSyncWithDestinationId(destinationConnection.getDestinationId());
+    final StandardSync sourceSync = ConnectionHelpers.generateSyncWithSourceId(sourceConnection1.getSourceId());
+    final StandardSyncSchedule syncSchedule = ConnectionHelpers.generateSchedule(sourceSync.getConnectionId());
+
+    // Read operations
+    when(configRepository.getStandardWorkspace(PersistenceConstants.DEFAULT_WORKSPACE_ID))
+        .thenReturn(workspace);
+    when(configRepository.listStandardSources())
+        .thenReturn(List.of(standardSource));
+    when(configRepository.listStandardDestinationDefinitions())
+        .thenReturn(List.of(standardDestination));
+    when(configRepository.listSourceConnection())
+        .thenReturn(List.of(sourceConnection1, sourceConnection2));
+    when(configRepository.listDestinationConnection())
+        .thenReturn(List.of(destinationConnection));
+    when(configRepository.listStandardSyncs())
+        .thenReturn(List.of(destinationSync, sourceSync));
+    when(configRepository.getStandardSyncSchedule(sourceSync.getConnectionId()))
+        .thenReturn(syncSchedule);
+
+    // Write operations
+    final ArgumentCaptor<StandardWorkspace> resultWorkspace = ArgumentCaptor.forClass(StandardWorkspace.class);
+    final ArgumentCaptor<StandardSourceDefinition> resultStandardSource = ArgumentCaptor.forClass(StandardSourceDefinition.class);
+    final ArgumentCaptor<StandardDestinationDefinition> resultStandardDestination = ArgumentCaptor.forClass(StandardDestinationDefinition.class);
+    final ArgumentCaptor<SourceConnection> resultSourceConnection = ArgumentCaptor.forClass(SourceConnection.class);
+    final ArgumentCaptor<DestinationConnection> resultDestinationConnection = ArgumentCaptor.forClass(DestinationConnection.class);
+    final ArgumentCaptor<StandardSync> resultSync = ArgumentCaptor.forClass(StandardSync.class);
+    final ArgumentCaptor<StandardSyncSchedule> resultSyncSchedule = ArgumentCaptor.forClass(StandardSyncSchedule.class);
+
+    migrationHandler.importData(migrationHandler.exportData());
+
+    verify(configRepository, times(1)).writeStandardWorkspace(resultWorkspace.capture());
```

Comment:
why do we need these argument captors? can't we just do? I think the argument captors here are redundant and we can just use the verify api. 

```
    verify(configRepository, times(1)).writeStandardWorkspace(workspace);
```
^ this will fail if `configRepository.writeStandardWorkspace()` is not called with the equivalent of the `workspace` object, which I think is what you are trying to test. It just avoids all of the argument captors.

Reply:
First time with mockito...

I do have to keep it for multiple calls, right?

Reply:
i don'tt hink so actually i think you can do
```
    verify(configRepository, times(1)).writeStandardWorkspace(workspace1);
    verify(configRepository, times(1)).writeStandardWorkspace(workspace2);
```

Reply:
ok cook, i will change in my next PR